lib/decode.js

```diff
@@ -0,0 +1,143 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+import {CborldError} from './error';
+import cbor from 'cbor';
+import {getCborldContextUrls} from './context';
+import {getTermCodecMap} from './codec';
+import util from 'util';
+
+/**
+ * Decodes a CBOR-LD byte array into a JSON-LD document.
+ *
+ * @param {object} options - The options to use when decoding CBOR-LD.
```

Comment:
I note that this is using `options` per convention, but elsewhere `args` is being used instead.  Such as `context.js`

Reply:
Old code was wrong, should be using `options` everywhere. Fixed in 7caeb40cbcbb3fc658a2db27dad61122d3fd779f.